Buildings/Controls/OBC/ASHRAE/PrimarySystem/ChillerPlant/Pumps/CondenserWater/Subsequences/Speed.mo

```diff
@@ -0,0 +1,222 @@
+﻿within Buildings.Controls.OBC.ASHRAE.PrimarySystem.ChillerPlant.Pumps.CondenserWater.Subsequences;
+block Speed
+  "Output design speed of condenser water pumps at current stage"
```

Comment:
I didn't see other sections in the RP-1711 specifying the design speed  of the dedicated CW pumps. 
However, in head pressure control sequence, section 5.2.10.4, item 2, it does specify how to control the pump speed. But it assumes the design speed is known.
I would assume this sequence outputs the design speed for both headed and dedicated pumps.

Reply:
The section you are mentioning also says that `Where condenser water pumps are dedicated, speed reset shall be independent for each chiller.`. I can draw 2 conclusions:
- In the 1711 July draft they are not considering the configuration with WSE and dedicated variable speed CW pumps (as there is no control diagram for such a configuration)
- If this sequence was to be applied to dedicated CW pumps, one could set the parameter values such that it would work, but I think there should be something in the documentation that would indicate this

I think it would be good to clarify this with Brandon, as not everyone will want to spend this much time on figuring out if this is a sequence to use in their particular case, especially since most of the document states the configuration (i.e. dedicated vs. headered) pretty clearly.